src/batch-client/index.ts

```diff
@@ -1457,16 +1457,18 @@ export class ZimbraBatchClient {
 			}
 		});
 
-	public resetPassword = ({ password, dryRun }: ResetPasswordOptions) =>
+	public resetPassword = ({ password, dryRun, getPasswordRules, cancelResetPassword }: ResetPasswordOptions) =>
 		this.jsonRequest({
 			name: 'ResetPassword',
 			namespace: Namespace.Account,
 			body: {
 				password,
-				dryRun
+				dryRun,
+				getPasswordRules,
+				cancelResetPassword
 			},
 			singleRequest: true
-		}).then(() => true);
```

Comment:
why did you removed this ?

Reply:
here we were returning a boolean value instead of actual response of server and Aayush changed that.

Reply:
earlier the api wasnt giving any response, so we were explicitly returning boolean, but now we have response, so that's why `then` block is removed.
for reset password case, we wont be passing `getPasswordRules`. Just the password attribute